### PR TITLE
Use centraldogma 0.57

### PR DESCRIPTION
**Pull request by mauhiz**
Contents
- Bump dependencies for CD, Jackson, SLF4J, Logback
- Fix deprecation warnings related to CD (which recently introduced CentralDogmaRepository)

After this PR, decaton will require CD v0.55+ ( https://github.com/line/centraldogma/releases )

### CODE REVIEW

**Comment by mauhiz** [comment]
ping maintainers ( @ocadaruma )

File: build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ subprojects {
     }
 
     ext {
-        slf4jVersion = "1.7.32"
+        slf4jVersion = "2.0.0"
```
Comment (ocadaruma) [inline build.gradle:36]: Seems `2.0.1` has been just published

**Review by ocadaruma — COMMENTED** [review]
Thanks! Sorry for delaying the review.

Left nits comment but overall looks good

**Review by ocadaruma — APPROVED** [review]

File: build.gradle (hunk 1 above)
Comment (mauhiz) [reply build.gradle]: Actually, I noticed it's not a great idea to use slf4j 2+ due to https://github.com/spring-projects/spring-boot/issues/12649 (current spring-boot versions cannot support it). Let me submit another PR to downgrade it to latest 1.X

**Pull request merged**